packages/test/test-end-to-end-tests/src/test/gc/gcLoadingFromOlderSummaries.spec.ts

```diff
@@ -178,327 +158,13 @@ describeNoCompat("GC loading from older summaries", (getTestObjectProvider) => {
 		// Summarize - summary3 with the new summarizer. Before it summarizes, it will catch up to latest and so the
 		// reference state of the data stores should be the same as in summary2.
 		// Also, note that while catching up, it will download summary2 and update state from it.
-		const summaryResult3 = await summarizeNow(summarizer2);
```

Comment:
I don't remember but does all of these tests have to do with refreshing from snapshot. It certainly looks that way but please make sure that's the case :-).

Reply:
Change reverted